vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/HypervisorHostHelper.java

```diff
@@ -198,7 +198,7 @@ public static String getPublicNetworkNamePrefix(String vlanId) {
         if (UNTAGGED_VLAN_NAME.equalsIgnoreCase(vlanId)) {
             return "cloud.public.untagged";
         } else {
-            return "cloud.public." + vlanId;
+            return "cloud.public." + vlanId + ".";
```

Comment:
@Pearl1594 would that be wrong or cause compatibility issues post upgrade? Generally port group names don't end in period (`.`) cc @PaulAngus @andrijapanicsb @borisstoyanov @vladimirpetrov?

Reply:
@rhtyd the naming convention as seen in the vcenter/mob is cloud.public.<primvlanid>.<secondaryvlanid>.<network_rate>.<vswitchname> so, I believe, it shouldn't cause any issue.

![image](https://user-images.githubusercontent.com/10495417/89760452-0c418d80-db0a-11ea-99e5-a0cfc17469dd.png)

Reply:
Ah okay, in that case this LGTM - but please also check where the method `getPublicNetworkNamePrefix` is used if it could cause any regressions?